### ISSUE TITLE
`FILE_UNORDERED_IMPORTS` false positive when the list of imports contains an alias import

### ISSUE THREAD

**Issue by 0x6675636b796f75676974687562** [saveourtool/diktat#1494]
Traditionally, _IDEA_ places all alias imports into a separate group which can't be removed:

![import-layout](https://user-images.githubusercontent.com/73111822/183693830-942d52e9-4c5b-4b85-a23b-5a77059889f5.png)

On the contrary, _diKTat_ doesn't distinguish between regular and alias imports and requires that all imports are sorted alphabetically.

As a workaround, the user has to suppress `FILE_UNORDERED_IMPORTS` whenever alias imports are used:

```kotlin
@file:Suppress("FILE_UNORDERED_IMPORTS")

package com.example

import java.lang.String

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.TestMethodOrder
import org.junit.jupiter.params.ParameterizedTest
import org.junit.jupiter.params.provider.ValueSource

import java.lang.Object as JvmObject
```

## Environment information
* diktat version: 1.2.3
